src/Core/Framework/DataAbstractionLayer/Entity.php

```
@@ -133,6 +100,10 @@ public function get(string $property)
         }
 
         if ($this->has($property)) {
+            if (!isset($this->$property)) { // to cover not set properties without default value
+                return null;
```

@reviewers please pay attention to this change, maybe discussion needed.

---

as discussed in the daily, imo we should not do something like this! It would hide errors. yes some tests would need adjustments, but in production every entity should be set up properly. if not, we would not get informed about that, because of this. and this would make the native strong type advantages dissolving

---

Fixed in https://github.com/shopware/shopware/pull/6561/commits/93cfce9542163642c454b5949d5aa17fc8da4dc7